doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -7,9 +7,9 @@ Verified version
 ****************
 
 - Ubuntu version: **18.04**
-- GCC version: **9.0**
-- ACRN-hypervisor branch: **release_2.0 (acrn-2020w23.6-180000p)**
-- ACRN-Kernel (Service VM kernel): **release_2.0 (5.4.43-PKT-200203T060100Z)**
+- GCC version: **7.4**
+- ACRN-hypervisor branch: **release_2.2 (acrn-2020w40.1-180000p)**
+- ACRN-Kernel (Service VM kernel): **release_2.2 (5.4.43-PKT-200203T060100Z)**
```

Comment:
Is this correct? The version I'm picking up from the `release_2.2` branch is some 5.4.59 version. This is from the `acrn-kernel` repo, or should I look at a different repo?

Reply:
Sorry for forget update kernel version.